### ISSUE TITLE
Can't join/leave challenges : Error request entity too large.

### ISSUE THREAD

**Issue by bregishneck** [HabitRPG/habitica#5997]
Every time I try to join a challenge I get an error report that says, "Error request entity too large." It told me to try reloading the page but that hasn't worked.

**Comment by crookedneighbor** [comment]
Which challenge are you trying to join? Please provide a link to the challenge if possible.

**Comment by KrishnaRSampath** [comment]
I'm likewise seeing this when I try to join this challenge with 300 participants: https://habitica.com/#/options/groups/challenges/d80aff5b-f43a-44c0-b8fa-a05ab4b6757c

I'm using Chrome 45 for Mac OSX 10.11

**Comment by crookedneighbor** [comment]
@KrishnaRSampath 

What's your User ID? You can find it in [Settings > API](https://habitrpg.com/#/options/settings/api). Be sure to only copy the User ID and _not_ your API Token.   

I can add you manually to the challenge.

**Comment by KrishnaRSampath** [comment]
User name: LordKrishna
User ID: 41ee880f-9ca0-44de-97c8-7e464d708565

**Comment by Sapokanikan** [comment]
Hello.

I'm getting the same error too. However it is only happening with one of the challenges out of the 4 or so that i joined.

Here is the [challenge](https://habitica.com/#/options/groups/challenges/1deb5c06-33a0-4d24-8edb-f911be82964c)

I tried to join yesterday, refreshed, tried again. I did this multiple times throughout the day and nothing changed. Tried again a few times today and I'm getting the same results.

I'm running
OS: Linux 4.2.3-1
Browser: Mozilla Firefox 41.0.1

Let me know if any more info would be helpful.

**Comment by crookedneighbor** [comment]
@Sapokanikan I can add you to the challenge if you post your unique user id. You can find it in [Settings > API](https://habitrpg.com/#/options/settings/api). Be sure to only copy the User ID and _not_ your API Token.

**Comment by Sapokanikan** [comment]
@crookedneighbor I'm really sorry for the late reply. 

I would really appreciate that, thank you.

Edit: removed user id

**Comment by crookedneighbor** [comment]
@Sapokanikan No worries. I've added you to the challenge.

**Comment by Sapokanikan** [comment]
@crookedneighbor Thanks!

**Comment by karibean** [comment]
I'm having the same problem. I managed to join other challenges on the same day, but it won't let me join the Tavern Reading challenge https://habitica.com/#/options/groups/challenges/238b8b02-ee52-4a99-933c-3ce938c125d8

user id: 302492bb-9955-4a18-889b-db0cf8de201f

**Comment by Alys** [comment]
@karibean That challenge was closed in the past few hours.

**Comment by Dewines** [comment]
@Alys Actually, that challenge had to be deleted by the owner because he couldn't get it to close due to that error. Also someone has just reported in the Tavern that, "I want to join the Fight Back Against Depression challenge, but I get an error message every time I try. Is there any way to work around this?" I tried to join to check whether the problem had resolved itself, but I got the same error. I did successfully join and then leave a much smaller challenge (70 participants) immediately afterwards and everything worked as it should.

**Comment by Auleliel** [comment]
@Dewines Ah, yes, that was me having trouble joining the Fight Back Against Depression challenge. I tried to join a similar challenge that only had 30 people (the #$%& You, Depression! challenge in The Chronic Illness Guild), but got the same error message.

**Comment by fghber** [comment]
Suffering from the same issue when I try to join the Challange "10 Habits You Should Do Every Day."
Can you add me manually to that challange or is it over or something?

**Comment by fghber** [comment]
Sorry, forgot my UID: 2379dd46-aaf9-4b95-80fe-cd89f62e5864

**Comment by danlitwin** [comment]
I, too, am having this error trying to join the ["I'm awesome"](https://habitica.com/#/options/groups/challenges/f5469002-b21d-4657-8450-2d194e846a73) challenge. (Which I thought I was actually in, but may have experienced this before and forgotten about it.) Please add me manually if necessary, or ask me anything that can help you track down the bug. Thanks!

User ID: 0569c2d4-f59d-4bab-a4e7-1df9c9e9e06d

**Comment by Alys** [comment]
@fghber We're having some trouble adding you to that challenge (eef51c7c-3d99-405a-a662-2bd4744f9b54) but we'll keep trying.

@danlitwin You should be in that challenge now. Let us know if that's not the case.

**Comment by kanabe** [comment]
Hi there! 

I posted in the other thread about this issue awhile back: https://github.com/HabitRPG/habitrpg/issues/1471 hoping someone would add me onto the challenge, but I didn't know another thread was made. I'm still having the same problem to join this challenge: https://habitica.com/#/options/groups/challenges/eef51c7c-3d99-405a-a662-2bd4744f9b54

I've refreshed, and tried joining it on Mozilla, Chrome, and Safari. Any help would be appreciated. Thank you!

My user ID is dc376f78-c16a-4807-a96d-2352417662b4
